pkg/odo/cli/describe/component.go

```diff
@@ -443,7 +463,11 @@ func listComponentsNames(title string, devfileObj *parser.DevfileObj, typ v1alph
 	}
 	log.Info(title)
 	for _, container := range containers {
-		log.Printf("%s", container.Name)
+		printmsg := container.Name
+		if container.Container != nil {
+			printmsg += fmt.Sprintf("\n    ProjectSource: %s", container.Container.SourceMapping)
```

Comment:
Per the acceptance criteria, I think it should not report this at all if `mountSources` if `false`. Currently, it is displaying an empty value for a container component where `mountSources` is `false`:

```
$ odo describe component                    
Name: a-test-nodejs
Display Name: Node.js Runtime
Project Type: Node.js
Language: JavaScript
Version: 2.2.5
Description: Stack with Node.js 16
Tags: Node.js, Express, ubi8

Running in: None

Supported odo features:
 •  Dev: true
 •  Deploy: false
 •  Debug: true

Container components:
 •  runtime
    ProjectSource: 
```

Reply:
For consistency with how the field is named in the JSON output, I'd suggest renaming this into `Source Mapping`. What do you think?

Reply:
>For consistency with how the field is named in the JSON output, I'd suggest renaming this into Source Mapping. What do you think?

I do like the idea of it being consistent, but I am not sure if `Source Mapping`  makes sense on UI. I guess I'll change that.

Reply:
>Per the acceptance criteria, I think it should not report this at all if mountSources if false. Currently, it is displaying an empty value for a container component where mountSources is false:

I thought it would be covered by the `updateWithRemoteSourceLocation` function, but perhaps you're right, I've added it.